### PR TITLE
add nft endpoints and entities

### PR DESCRIPTION
**Pull request by idanya**

### CODE REVIEW

File: src/fireblocks-sdk.ts [hunk 1]
```diff
@@ -1150,4 +1153,47 @@ export class FireblocksSDK {
...
+     */
+    public async getNFT(tokenId: string): Promise<Token> {
+        return await this.apiClient.issueGetRequest(`/v1/nfts/tokens/${tokenId}`);
+    }
+
+    /**
+     *
+     * @param tokenIds comma seperated list of token ids to fetch
+     */
+    public async getNFTs(tokenIds: string): Promise<Token[]> {
```
Comment (or109) [inline src/fireblocks-sdk.ts:1169]: WDYT for having array of string here?

```suggestion
    public async getNFTs(tokenIds: string[]): Promise<Token[]> {
```

File: src/fireblocks-sdk.ts [hunk 2]
```diff
@@ -1150,4 +1153,47 @@ export class FireblocksSDK {
...
+        return await this.apiClient.issueGetRequest(`/v1/nfts/tokens?ids=${tokenIds}`);
+    }
+
+    /**
+     *
+     * @param filter filter results object
+     */
+    public async getOwnedNFTs(filter?: NFTOwnershipFilter): Promise<TokenWithBalance[]> {
+        return await this.apiClient.issueGetRequest(`/v1/nfts/ownership/tokens?${queryString.stringify(filter)}`);
```
Comment (or109) [inline src/fireblocks-sdk.ts:1178]: If we decide to go with an array of strings for ids
Should it be more explicit and not use stringify?

BTW, for which params the `stringify` is needed?

File: src/fireblocks-sdk.ts [hunk 3]
```diff
@@ -1150,4 +1153,47 @@ export class FireblocksSDK {
...
+    }
+
+    /**
+     *
+     * @param vaultAccountId
+     * @param blockchainDescriptor
+     */
+    public async refreshNFTOwnershipByVault(vaultAccountId: string, blockchainDescriptor: string): Promise<void> {
+        return await this.apiClient.issuePutRequest(
+            `/v1/nfts/ownership/tokens?vaultAccountId=${vaultAccountId}&blockchainDescriptor=${blockchainDescriptor}`,
```
Comment (or109) [inline src/fireblocks-sdk.ts:1196]: Let's keep it consistent with other usages

File: src/types.ts [hunk 4]
```diff
@@ -478,6 +478,47 @@ export interface TransactionFilter {
...
+}
+
+export interface TokenWithBalance extends Token {
+    balance: TokenBalance;
+}
+
+class MediaEntity {
+    url: string;
+    contentType: string;
+    originalURL?: string;
```
Comment (or109) [inline src/types.ts:499]: Let's check about this param if we expose it or not

**Review by or109 — COMMENTED** [review]
Small comments

**Review by chenshoo — APPROVED** [review]

File: src/fireblocks-sdk.ts [hunk 5]
```diff
@@ -53,6 +53,9 @@ import {
     NetworkIdResponse,
     TimePeriod,
     AuditsResponse,
+    NFTOwnershipFilter,
+    Token,
+    TokenWithBalance, APIPagedResponse
```
Comment (or109) [inline src/fireblocks-sdk.ts:58]: Yalla linebreak

**Comment by or109** [comment]
Looking ✅

**Pull request merged**